x-pack/test/functional/apps/dashboard/migration_smoke_tests/lens_migration_smoke_test.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* This test is importing saved objects from 7.13.0 to 8.0 and the backported version
+ * will import from 6.8.x to 7.x.x
+ */
+
+import expect from '@kbn/expect';
+import path from 'path';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+  const testSubjects = getService('testSubjects');
+  const dashboardPanelActions = getService('dashboardPanelActions');
+
+  const PageObjects = getPageObjects(['common', 'settings', 'header', 'savedObjects', 'dashboard']);
+
+  describe('Export import saved objects between versions', () => {
+    before(async () => {
+      await esArchiver.loadIfNeeded(
+        'x-pack/test/functional/es_archives/getting_started/shakespeare'
+      );
+      await kibanaServer.uiSettings.replace({});
+      await PageObjects.settings.navigateTo();
+      await PageObjects.settings.clickKibanaSavedObjects();
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/getting_started/shakespeare');
+      await esArchiver.load('x-pack/test/functional/es_archives/empty_kibana');
+    });
+
+    it('should be able to import dashboard with various Lens panels from 7.12.1', async () => {
+      await PageObjects.savedObjects.importFile(
```

Comment:
I spoke with @tylersmalley about how we might load legacy saved object data into Kibana and he mentioned we should use the `kbnArchiver` which is a tool that aims to replace `esArchiver` and other methods for loading of saved objects. Example usage:

https://github.com/elastic/kibana/blob/38604863e593e9c7aa239d077a1317709ac50131/test/api_integration/apis/saved_objects/bulk_update.ts#L19

This will also interact with Kibana via the API rather than the UI to load data which should be faster. Let me know what you think!

Reply:
I will look into this! Anything that speeds up the run is a good thing. I assume that this can support `ndjson` files?

Reply:
In speaking to @tylersmalley, it like it doesn't support `ndjson`, and I would need to load up 7.12 again and re-export. It's probably worth mentioning this in the description so that people know it's an option going forward, but I don't think it's necessary to update to this format on this PR.

Reply:
I have an open issue to add support for a pretty format to the saved object export which would allow us to directly support that format: https://github.com/elastic/kibana/issues/103021